src/thenP.js

```diff
@@ -0,0 +1,27 @@
+import { invoker } from 'ramda';
+
+/**
+ * Composable shortcut for `Promise.then`.
+ * The thenP returns a Promise. It takes two arguments: callback function for the success of the Promise
+ * and the promise instance itself.
+ *
+ * @func thenP
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.8.0|v2.8.0}
+ * @category Function
+ * @sig (a -> Promise b | b) -> Promise b
+ * @param {Function} onFulfilled A Function called if the Promise is fulfilled. This function has one argument, the fulfillment value
+ * @param {Promise} promise Any Promise or Thenable object
+ * @return {Promise} A Promise in the pending status
```

Comment:
A Promise in pending state

Reply:
Copied directly from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then